components/Pages/Payruns/PayrunList.js

```diff
@@ -36,7 +36,7 @@ export const PayrunList = ({ searchTerm, data }) => {
                 <Container display="flex" alignItems="center">
                   <Heading size="s">Payrun ID: </Heading>
```

Comment:
need to change this text here, ask burak for what to change it to

Reply:
Payrun Number
probably

Reply:
true